### DISCUSSION TITLE
Getting "Error running Spectral!" when I add "extends: spectral:oas" in spectral.yaml

### DISCUSSION THREAD

**Discussion by virajsathvara-oa** [stoplightio/spectral#2169] [Q&A]
So I am migrating our spectral rules from V5 to V6 but keep getting  *Error running Spectral!* in console when I run @stoplight/spectral-cli via command prompt. 

I am doing spectral lint openapi.yaml --ruleset spectral.yaml --verbose.

Funny thing I noticed is that when I remove *extends: spectral:oas* from the top of my yaml, the ruleset works. 

Anyone knows how to tackle this?

**Comment by philsturgeon** [comment]
Is the the entirety of the output or is there something else?
